src/app/components/toggle.tsx

```diff
@@ -0,0 +1,50 @@
+import { css } from '@emotion/react';
+import * as Switch from '@radix-ui/react-switch';
+import { useField } from 'formik';
+
+interface ToggleProps {
+  name: string;
+}
+
+export function Toggle({ name }: ToggleProps) {
+  const [field, , helpers] = useField(name);
```

Comment:
I would separate these into two components:

- Toggle, that is just the bare minimium component that accepts state like `on|off`
- ToggleInput, that contains `useField` formik specific hooks